### DISCUSSION TITLE
the stable/prometheus and stable/grafana helm charts repo are deprecated

### DISCUSSION THREAD

**Discussion by ssam13** [Kong/kubernetes-ingress-controller#1827] [Q&A] [answered]
Hi everybody!
I want to monitor my `kong/kubernetes-ingress-controller` with `prometheus `and `grafana`.
in the documentaion of KIC they did this using `helm `, but the problem is helm charts of `prometheus `and `grafana `used in the installation guide are **DEPRECATED**.
Can you help me with what I should do about this problem?
Thanks!

**Comment by shaneutt** [comment, marked as answer]
Hi @hesmoh :wave: 

Yes the guide is using deprecated Prometheus components and should be updated. We would welcome you to put in an issue for this in https://github.com/kong/docs.konghq.com as we will need to get on top of replacing these soon.

In the meantime you may want to consider the Prometheus Operator:

- https://prometheus-operator.dev
- https://github.com/prometheus-operator

And use these in place of the deprecated deployment methods.

**Reply by ssam13** [reply]
thanks @shaneutt
